src/api/app/views/webui2/shared/_buildresult_box.html.haml

```diff
@@ -0,0 +1,91 @@
+:ruby
+  index ||= ''
+  if defined?(package)
+    controller = 'package'
+  else
+    controller = 'project'
+  end
+  ajax_data = {}
+  ajax_data['project'] = h(project) if defined?(project)
+  ajax_data['package'] = h(package) if defined?(package)
+  ajax_data['index'] = h(index) if defined?(index)
+
+.box-header.header-tabs
+.card.mb-3
+  .bg-light
+    %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible#buildresult-box{ role: 'tablist', data: ajax_data }
```

Comment:
That's the only thing I don't like about it, visually. I guess I could live with smaller or less text...

![screenshot from 2018-08-22 16-49-13](https://user-images.githubusercontent.com/514785/44471070-6e5a0d00-a62b-11e8-8626-57d00c74af6c.png)

Reply:
I would give up on text with "failed", "building", "finished" and "succeeded" and instead made it [where: fa icon (bootstrap colour)]: times (danger), tasks (info), check (secondary) and check (success). Also lorry should very much be coloured because it's hard to understand its status from it currently.

To get better and more consistent colouring of stripped tables out of bootstrap, embeding a table inside a table should do that (otherwise distros next to each other might have the same colour if the number of sublines [arch lines] is uneven). That takes some more tinkering with it, but in the end it might be worth the effort so it's easier to read.

I would shorten those tab names to something simpler because assuming this is an xl bootstrap breakpoint, it's not gonna look any better on lower ones :D 
Results and RPM Lint should be understandable enough.

Actually speaking of RPM Lint, I heard compliant that part is not visible enough and maybe incorporating some kind of pill badge in there with a bright red background might be a good idea (or if you are adventurous enough, maybe even red badge for error and yellow badge for warning :o) 

Out of curiosity, maybe repository name can have some kind of human readable name version without all those ugly underscores in there (and underscores are usually longer than spaces, so another measure to leave more space in there for even longer names).

Reply:
We made some changes in buildresult box:

![screenshot_2018-09-04 show home admin perl-citrix - open build service 1](https://user-images.githubusercontent.com/1212806/45032876-25ef1600-b053-11e8-86c7-760e0e1c7c5d.png)

> I would give up on text with "failed", "building", "finished" and "succeeded" and instead made it [where: fa icon (bootstrap colour)]: times (danger), tasks (info), check (secondary) and check (success). Also lorry should very much be coloured because it's hard to understand its status from it currently.

I prefer to keep the text and icons, because sometimes they are not self-explanatory, for examples `unresolvable`.